_topic_maps/_topic_map.yml

```diff
@@ -293,6 +293,8 @@ Topics:
     File: connect-trigger-sink
   - Name: Filtering triggers from the command line
     File: filter-triggers-cli
+  - Name: New trigger filters
+    File: new-trigger-filters
```

Comment:
Same here, I'd call this "Advanced trigger filters"

Reply:
Changed the name